CLAUDE.md

```
@@ -0,0 +1,75 @@
+npm run lint:fix       # biome check --fix src/
+npm run format         # biome format --write src/
+npm run check          # lint + tsc --noEmit (run before committing)
+npm test               # vitest run
+npm run test:coverage  # vitest run --coverage
+```
+
+**Linter/formatter is Biome, not ESLint/Prettier.** Never add ESLint config.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)